dashboard/app/models/section.rb

```diff
@@ -15,17 +14,18 @@
 #  deleted_at   :datetime
 #  stage_extras :boolean          default(FALSE), not null
 #  section_type :string(255)
+#  user_id      :integer
```

Comment:
Note that the migration drops this column. Does it make sense to preserve its ordering (e.g., after `id`) within the columns?